schema/commonmark.schema.json

```diff
@@ -525,8 +525,7 @@
       "required": ["url"],
       "properties": {
         "url": {
-          "type": "string",
-          "format": "uri-reference"
```

Comment:
This one test is still a link, despite incorrect uri formatting: https://spec.commonmark.org/0.30/#example-506